app/commands/thredded/notify_following_users.rb

```diff
@@ -6,13 +6,13 @@ def initialize(post)
     end
 
     def run
-      return unless targetted_users.present?
-      PostMailer.post_notification(@post.id, targetted_users.map(&:email)).deliver_now
-      MembersMarkedNotified.new(@post, targetted_users).run
+      return if targeted_users.empty?
+      PostMailer.post_notification(@post.id, targeted_users.map(&:email)).deliver_now
+      MembersMarkedNotified.new(@post, targeted_users).run
```

Comment:
How does this in particular open email addresses to others?  could we change this to BCC: instead of TO: